src/ui/public/styles/spinner.less

```diff
@@ -1,39 +1,6 @@
-@import "~ui/styles/variables";
-
-@size: 400px;
-@color1: @kibanaPink1;
-@color2: @kibanaPink2;
-
 .spinner.ng-hide {
```

Comment:
ah, I should have looked for double quotes as well. I was confusing style guides. Doh!